crates/bitwarden-json/src/command.rs

```diff
@@ -32,7 +35,7 @@ pub enum Command {
     ///
     /// This command is not capable of handling authentication requiring 2fa or captcha.
     ///
-    /// Returns: [PasswordLoginResponse](crate::sdk::auth::response::PasswordLoginResponse)
+    /// Returns: [PasswordLoginResponse](bitwarden::auth::response::PasswordLoginResponse)
```

Comment:
Can you please move all of these changes into a separate PR?

Reply:
Moved into https://github.com/bitwarden/sdk/pull/80